includes/class-wp-job-manager-post-types.php

```diff
@@ -702,6 +703,9 @@ public function job_feed_item() {
 		if ( $company ) {
 			echo '<job_listing:company><![CDATA[' . esc_html( $company ) . "]]></job_listing:company>\n";
 		}
+		if ( $salary ) {
+			echo '<job_listing:salary><![CDATA[' . esc_html( $salary ) . "]]></job_listing:salary>\n";
+		}
```

Comment:
Couldn't understand what's this used for.

Reply:
Not sure either but I think it has something to do with RSS feeds